pyrogram/methods/messages/forward_messages.py

```diff
@@ -47,9 +47,8 @@ async def forward_messages(
                 For your personal cloud (Saved Messages) you can simply use "me" or "self".
                 For a contact that exists in your Telegram address book you can use his phone number (str).
 
-            message_ids (``int`` | List of ``int``):
+            message_ids (``int`` | ``List[int]``):
```

Comment:
The reason behind "List of Type" is to keep clickable links in case of "List of Message" (Message is a link), for example. For consistency this was kept for bare types as well.

Reply:
should I change all List[Type] to "List of Type"?

Reply:
I've to write them like this?
List of ``str`` | List of ``int``